docs/docs/developer-guides/cosmwasm/walkthrough.md

```diff
@@ -150,3 +150,42 @@ ccw substrate -c http://127.0.0.1:32200 \
     -g 100000000 \
     -q '{"balance":{"address":"0xd64439add16b49b6b68ac74e1b28a73a8491501ab7e0e829716f580947a4bd7e"}}'
 ```
+
+## Running `pallet-cosmwasm` on Picasso Rococo
+
+This is a specific example guide to upload, initialize and execute cw20 contracts on Picasso Rococo. The previous walkthough can also be applied to Picasso Rococo too, however, we have added another example specific to Rococo.
+
+
+```sh
+# Download the contract
+curl --location https://github.com/CosmWasm/cw-plus/releases/download/v1.0.1/cw20_base.wasm > cw20_base.wasm`
+```
+```sh
+# Upload the contract 
+cargo run substrate -c wss://picasso-rococo-rpc-lb.composablenodes.tech:443 --seed "<SEED>" tx upload --file-path ./cw20_base.wasm 
+```
+```sh
+# Instantiate the contract* 
+cargo run substrate -c wss://picasso-rococo-rpc-lb.composablenodes.tech:443 --seed "<SEED>" tx instantiate --code-id 1 --salt 0x1234 --label 0x4321 --gas 10000000000 --message '{ "decimals" : 18, "initial_balances": [], "name" : "SHIB", "symbol" : "SHIB", "mint": {"minter" : "5yNZjX24n2eg7W6EVamaTXNQbWCwchhThEaSWB7V3GRjtHeL"} }'
+```
+```sh
+# Execute the contract
```

Comment:
add section about checking correct contract address

Reply:
I added where the * is

Reply:
You say there about minter address, not contract address